docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,107 @@
+---
+title: ".NET docs what's new (November 2019)"
+description: "What's new in .NET docs in November 2019."
+ms.date: 12/16/2019
+---
+
+# .NET docs what's new (November 2019)
+
+Welcome to what's new in .NET docs for November 2019. Listed below are some of the recent docs we've published.
+
+## .NET Core
+
+- ***new!*** [Breaking changes for migration from Version 2.2 to 3.1](../core/compatibility/2.2-3.1.md) - Core breaking changes improvements
```

Comment:
I'm sorry if this was already answered, but what is considered ***new!***? Does that mean brand new instead of newly updated?

Reply:
For "new', a PR added the file.